packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketLayoutNavigation.tsx

```diff
@@ -36,19 +36,25 @@ const NavListItem = styled(NavigationItem)`
             : ''}
 `;
 
-const Divider = styled.div`
+const StyledDivider = styled.div<{ elevation: Elevation }>`
     margin: ${spacingsPx.xxs} 0; /* gap xxs + margin xxs = spacing xs around divider */
     width: 100%;
-    border-bottom: 1px solid ${({ theme }) => theme.borderOnElevation0};
+    border-bottom: 1px solid ${({ theme, elevation }) => theme[mapElevationToBorder[elevation]]};
```

Comment:
Same as this one. It will never be on a different elevation level, will it? 🤔

Reply:
I like to make it strict, the direct elevation shall not be used anywhere. 

Its not just this is local, but its parent component can be put in the card or something else what changes elevation and then the color shall be changed as well. If its hardcoded you have to change it as well

Reply:
I don't see the benefit to be honest and I don't think it should be super strict. The component is named `CoinmarketLayoutNavigation.tsx` and it is used in 1 place in the app and never anywhere else, nor it should not be use anywhere else. 

Elevation context should be used only for components which can exist in multiple elevations, otherwise it introduces unnecessary complexity which defeats it's whole purpose 🙏